samba: build dnsserver_common code

Just 'install' does not seem to do it.

Upstream-Status: Pending

Signed-off-by: Joe Slater <joe.slater@windriver.com>

--- a/source4/dns_server/wscript_build
+++ b/source4/dns_server/wscript_build
@@ -4,7 +4,7 @@ bld.SAMBA_LIBRARY('dnsserver_common',
         source='dnsserver_common.c',
         deps='samba-util samba-errors ldbsamba clidns',
         private_library=True,
-        install=bld.AD_DC_BUILD_IS_ENABLED()
+        enabled=bld.AD_DC_BUILD_IS_ENABLED()
         )
 
 bld.SAMBA_MODULE('service_dns',
